config/karma/shared.karma.conf.js

```diff
@@ -10,11 +10,10 @@ function getConfig(config) {
   // This file is spawned so we'll need to read the args again
   const minimist = require('minimist');
   const argv = minimist(process.argv.slice(2));
-
   const path = require('path');
   let testWebpackConfig = require('../webpack/test.webpack.config');
   let remapIstanbul = require('remap-istanbul');
-  let skyPagesConfig = require('../sky-pages/sky-pages.config').getSkyPagesConfig(argv.command);
+  let skyPagesConfig = require('../sky-pages/sky-pages.config').getSkyPagesConfig(argv._[0]);
```

Comment:
Can you explain this change?  Looking at the code, this seems like a bug in builder as `argv.command` would never exist since the config file is having to read form the command line again.  Can you elaborate on it?

Reply:
Throwing a `console.log` right before that line gives `{ _: [ 'pact' ] }`, and I checked it for existing commands like `skyux test` and it gives the same output.  I noticed that and I changed it, I can test on a fresh copy of builder to be sure it's not related to any of my changes.

Reply:
Alright, on a clean version it's still `{ _: [ 'test' ] }`.  I believe this is because argv is re-initialized in `shared.karma.conf.js` after `argv.command` is set in the cli command file.

Reply:
Well good catch.  This is a bug in builder that I'll plan on fixing outside the context of this PR.